monkey/monkey_island/cc/services/telemetry/zero_trust_tests/data_endpoints.py

```diff
@@ -31,7 +32,18 @@ def test_open_data_endpoints(telemetry_json):
             event_type=EVENT_TYPE_ISLAND
         ))
         if service_name in HTTP_SERVERS_SERVICES_NAMES:
-            found_http_server_status = STATUS_CONCLUSIVE
+            found_http_server_status = STATUS_FAILED
+            events.append(Event.create_event(
+                title="Scan telemetry analysis",
+                message="Service {} on {} recognized as an open data endpoint! Service details: {}".format(
+                    service_data["display_name"],
+                    telemetry_json["data"]["machine"]["ip_addr"],
+                    json.dumps(service_data)
+                ),
+                event_type=EVENT_TYPE_ISLAND
+            ))
+        if service_name in 'elastic-search-9200':
```

Comment:
Export to const (and hopefully someday start moving some consts to common to prevent code duplication)

Reply:
Done